com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/UnlitPass.hlsl

```diff
@@ -1,13 +1,46 @@
-﻿PackedVaryings vert(Attributes input)
+﻿
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl"
+
+void InitializeInputData(Varyings input, out InputData inputData)
```

Comment:
I guess we need to investigate this again. However we should really consider moving to elsewhere as this does not look right in lighting.hlsl

Reply:
This relies on us being able to split out all of the debugging code from "Lighting.hlsl" - currently we can do this because it relies on certain lighting methods (for access to lights, shadow-cascades and BRDF data). We would need to refactor this code in order to do this - maybe split out the light code into one file, shadow code into another and BRDF out into a third. @arttu-peltonen